README.md

```diff
@@ -122,6 +124,8 @@ Help / Documentation
 
 - API reference documentation: [![Read the documentation of the stable version](https://readthedocs.org/projects/pip/badge/?version=stable)](http://bqplot.readthedocs.org/en/stable/) [![Read the documentation of the development version](https://readthedocs.org/projects/pip/badge/?version=latest)](http://bqplot.readthedocs.org/en/latest/)
 
+- Talk to us on the Gitter chat: [![Join the chat at https://gitter.im/bloomberg/bqplot](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/bloomberg/bqplot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Just that I am already in more than 30 gitter rooms and I have less chances to miss things if widgets related things are together (and posts on bqplot are rare anyway...) Up to you.